packages/colorpickers/src/elements/ColorSwatch/index.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { Tooltip } from '@zendeskgarden/react-tooltips';
+import { useGrid } from '@zendeskgarden/container-grid';
+import { useId } from '@zendeskgarden/container-utilities';
+import { StyledSwatchButton, StyledCheckIcon, StyledCell, StyledColorSwatch } from '../../styled';
+
+export interface ILabeledColor {
+  value: string;
+  label: string;
+}
+
+export interface IColorSwatchProps {
+  /** Enables wrapped keyboard navigation */
+  isWrapped?: boolean;
```

Comment:
I think there's a case to be made that it remains off by default at the lower level hook, since wrapped navigation is an enhancement (and is optional in ARIA). It feels like opinions should be layered on at the higher levels (such as `react-components`).

At the component level, if Garden assumes the most common usage of `ColorSwatch` is many color chips (many rows, many columns), then a default wrapped navigation provides a more efficient navigation out of the box. For consumers using a single row & low column count color swatch, they make want unwrapped behavior, and can turn it off (which would be on by default).

Reply:
> I think there's a case to be made that it remains off by default at the lower level hook

Agreed. My comment was about setting a `true` value on the hook usage here in the `ColorSwatch`. I think the navigational behavior should be consistent (imo, wrapped) regardless of the chip count.